XCUITests/TopTabsTest.swift

```diff
@@ -63,7 +63,7 @@ class TopTabsTest: BaseTestCase {
         waitforExistence(app.webViews.links.staticTexts["More information..."])
         app.webViews.links.staticTexts["More information..."].press(forDuration: 5)
         app.buttons["Open in New Tab"].tap()
-        app.buttons["Switch"].tap()
+        sleep(2)
```

Comment:
I know adding sleep is not the best solution, but since the Switch button appears for a few seconds could cause false failures. The test was failed but it was possible to see the button, it was just timing issue not to be able to tap on it. If there is not delay between opening new tap and going to check the content of each tab, there is no time the second one is loaded, that is why adding this time here could make the test more stable. But if there is a better solution, please let me know :)!